indigo_app/templates/place/tabbed_layout.html

```diff
@@ -58,7 +58,7 @@
           </a>
         {% endif %}
         {% if user_can_change_place_settings %}
-          <a class="nav-item nav-link {% if view.tab == 'place_settings' %}active{% endif %}" href="{% url 'place_settings' place=place.place_code %}">
+          <a class="nav-item nav-link {% if view.tab == 'place_settings' or view.tab == 'place_users' %}active{% endif %}" href="{% url 'place_settings' place=place.place_code %}">
```

Comment:
the tab should be `place_settings` in both cases, now, I think?

Reply:
@longhotsummer the settings tab now serves two views, the place_users, and place_settings, we also need the separate tab names to activate the menu on the left.

Reply:
ah, of course